src/LinuxTracing/LinuxTracingUtils.cpp

```diff
@@ -213,4 +222,121 @@ bool SetMaxOpenFilesSoftLimit(uint64_t soft_limit) {
   return true;
 }
 
+// Check for the existence of a file mapping containing the absolute address of the function. We
+// have to consider the possibility that the module be mapped multiple times, and hence that the
+// function have multiple absolute addresses.
```

Comment:
Hehe, it was subjunctive, on purpose because it follows "the possibility that". I guess nobody uses it anymore... I switched to adding "may" to avoid confusion.